app/services/db/members-history.ts

```diff
@@ -105,7 +105,7 @@ export async function buildMembersHistory(requests: RequestWithData[]) {
 
       if (filteredNewRoles.length > 0) {
         newRoles = filteredNewRoles;
-      } else if (isNewUser || filteredNewRoles.length === 0) {
+      } else if ((isNewUser || filteredNewRoles.length === 0) && request.changes?.parentPaths.includes('members')) {
```

Comment:
I think this is fine, but I don't have enough context to know for sure.

For my own understanding would you mind explaining: 

- where does the `parentPaths` comes from? I'm assuming that `changes` is a field from the db that contains some kind of structure that includes `parentPaths`? Is that correct?
- is `members` a common value used in various places? I'm wondering about using a constant vs a string to avoid typo mistakes
- will `members` always be lower case? Does the code need to consider the string's case?